core/src/main/java/com/github/mizool/core/rest/errorhandling/behavior/UnprocessableEntityExceptionBehavior.java

```diff
@@ -13,7 +13,7 @@
     @Override
     public Class<? extends Throwable> getThrowableClass()
     {
-        return UnprocessableEntityException.class;
+        return AbstractUnprocessableEntityException.class;
```

Comment:
If you didn't do so already, please double-check that this indeed still works for the `UnprocessableEntityException` subclass. (I'm not sure whether we already had this use case.)

Reply:
Test com.github.mizool.core.rest.errorhandling.TestErrorHandlingBehaviorCatalog#whitelistedBySuperclass proves it works